src/server_manager/electron_app/index.ts

```diff
@@ -74,7 +74,6 @@ function createMainWindow() {
     webPreferences: {
       nodeIntegration: false,
       preload: path.join(__dirname, 'preload.js'),
-      nativeWindowOpen: true,
```

Comment:
The reason for this was to open links in the user's browser. Is that behavior presserved?

Reply:
Yes.  `nativeWindowOpen: false` is no longer supported, so the option has been removed.